modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/incubus_draft.dm

```diff
@@ -135,66 +137,118 @@
 	return ..()
 
 /datum/reagent/drug/aphrodisiac/incubus_draft/overdose_effects(mob/living/carbon/human/exposed_mob)
```

Comment:
This proc is literally over 100 lines long, it's absolute hell to review. Could you break it up into smaller procs that are called from this one instead, like `update_X()` and/or `grow_X()` and/or `remove_X()` where `X` is the name of the affected genital? I think it would make this whole proc a lot easier to read, and to maintain in the future.

For instance, you could add some early returns to massively reduce the indentation, if each of your big `if()` statements related to a specific genital was in its own proc instead.